test/Sentry.AspNetCore.TestUtils/Sentry.AspNetCore.TestUtils.csproj

```diff
@@ -51,8 +51,8 @@
   <ItemGroup Condition="$(TargetFramework) == 'net9.0'">
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="5.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="9.0.0" />
-    <PackageReference Include="Verify.AspNetCore" Version="4.0.0" />
-    <PackageReference Include="Verify.Http" Version="6.4.1" />
+    <PackageReference Include="Verify.AspNetCore" Version="4.2.0" />
+    <PackageReference Include="Verify.Http" Version="6.5.0" />
```

Comment:
question: Could/Should we update some `Verify.` versions for the above `<ItemGroup Condition="$(TargetFramework) == 'net8.0'">` as well?

Do I get that right?
- Both latest `Verify.` versions target `net9.0` only, so we need `net8.0` compatible versions above.

For above `Condition="$(TargetFramework) == 'net8.0'"`
may we update `Verify.AspNetCore` from `3.4.1` to `4.0.0`?
- `3.4.1` targets `net7.0`
- `4.0.0` targets `net8.0`
- `4.2.0` targets `net9.0`

For above `Condition="$(TargetFramework) == 'net8.0'"`
may we update `Verify.Http` from `4.3.2` to `6.4.1`?
- `4.3.2` targets `net6.0` and `net48`
- `6.4.1` targets `net8.0`
- `6.5.0` targets `net9.0`

Reply:
> Could/Should we update some Verify. versions for the above

The only reason I bumped Verify in this PR was to work out whether Verify was part of the problem. To some extent it was since the older version of Verify was hiding the real error but it's looking more like the latest version of .NET is compiling IL that the current version of Mono doesn't know how to handle. 

For this PR then, there's no reason to fiddle with net8.0 specific dependencies. The tests that are failing are all doing so on the net48 (mono) targets.